### PR TITLE
Use semantic HTML elements for Git version command

### PR DESCRIPTION
**Pull request by ugultopu**
While keeping selecting the whole command on click.

### CODE REVIEW

File: app/assets/stylesheets/reference.scss [hunk 1]
```diff
@@ -136,14 +136,21 @@ h3.plumbing {
...
+      all: unset;
+      display: block;
+      color: $light-font-color;
+    }
+
+    code {
+      user-select: all;
+    }
+
+    span {
```
Comment (peff) [inline app/assets/stylesheets/reference.scss:151]: A bare `span` seems a bit broad here (even though it is scoped to the dropdown). Would we want `span.prompt` or something?

**Review by peff — APPROVED** [review]
Thanks! I love that you were able to keep the selection working the same, and the results look good to me. One minor nit below that might be worth addressing.

File: app/assets/stylesheets/reference.scss (hunk 1 above)
Comment (ugultopu) [reply app/assets/stylesheets/reference.scss:151]: That makes sense actually. I figured there is only one place with ID `previous-versions-dropdown`, hence no need to go more specific, but adding a class would improve readability as well.

Comment (ugultopu) [reply app/assets/stylesheets/reference.scss]: Done!

**Comment by ugultopu** [comment]
[An even better solution](https://github.com/ugultopu/git-scm.com/commit/45ace2f80a20b6b10db22bc8c1d9f07ea1f4cd63)

**Comment by peff** [comment]
@ugultopu Yeah, that is nice, provided we do it with a class `<code class="example-command">` or something. If you think it's worth doing, open up a PR.

**Pull request merged**